cn_indian_payroll/cn_indian_payroll/patches/category.py

```
@@ -0,0 +1,143 @@
+import frappe
+
+def execute():
+    data=[
+ {
+  "docstatus": 0,
+  "doctype": "Employee Tax Exemption Category",
+  "is_active": 1,
+  "max_amount": 0.0,
+  "modified": "2024-12-10 11:54:02.651466",
+  "name": "Section 16(iii)"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)